README.md

```diff
@@ -102,15 +102,15 @@ To do this, run the following:
 # Use the cfgov-refresh virtualenv.
 workon cfgov-refresh
```

Comment:
Should `workon` come after `cd`'ing into the project directory or does it not matter?